roles/init_dbserver/templates/postgresql.conf.template

```diff
@@ -33,3 +33,8 @@ autovacuum_vacuum_cost_limit = 3000
 idle_in_transaction_session_timeout = 10min
 shared_preload_libraries = '{{ pg_shared_libraries_list | join(",") }}'
 cluster_name = '{{ pg_instance_name }}'
+{% if pg_init_conf_params|length > 0 %}
+{% for item in pg_init_conf_params %}
+{{ item.name }} = {{ item.value }}
```

Comment:
I'd recommend quoting the value here:
```yaml
{{ item.name }} = '{{ item.value }}'
```

Reply:
Wouldn't it treat everything, including boolean/number, as a string if we use quotes? This may lead to PostgreSQL startup failures.

Reply:
Fixed